src/applications/coronavirus-screener/routes.jsx

```diff
@@ -2,6 +2,6 @@ import React from 'react';
 import { Route } from 'react-router';
 import App from './containers/App.jsx';
 
-const routes = <Route path="/*" component={App} />;
+const routes = <Route path="/:id" component={App} />;
```

Comment:
Confirming that the naked route (/covid19screen/) will not be impacted?

Reply:
it does if https://github.com/department-of-veterans-affairs/vets-website/blob/8ae0809076efa46f2de3f9d81bbae33a466fb9f7/src/applications/coronavirus-screener/tests/00.coronavirus-screener.e2e.spec.js passes! (btw nightwatch e2e tests run only in jenkins now)